src/PostgREST/Types.hs

```diff
@@ -54,12 +54,12 @@ data PrimaryKey = PrimaryKey {
 } deriving (Show, Eq)
 
 data OrderDirection = OrderAsc | OrderDesc deriving (Eq)
-instance Show OrderDirection where
+instance S.Show OrderDirection where
```

Comment:
Protolude does not define a new typeclass, it only defines a polymorphic show function and does not expose the `show` from Prelude.
The only reason I've used this qualified was to be able to define new instances of Show since it is not imported in protolude.
But since then I've inspected the `GHC.Show` module and realized that would be silly to qualify it, since it does not conflict with anything, so I'm adding a commit to do that.
We can add GHC.Show to our custom Prelude later on if we want.